website/docs/contributing.md

```diff
@@ -150,4 +140,4 @@ git push --force-with-lease <remote-name> <branch-name>
 
 ## Code of Conduct
 
-This project has adopted the [Contributor Covenant Code of Conduct](./code-of-conduct.md).
\ No newline at end of file
+This project has adopted the [CNCF Code of Conduct](../../CODE_OF_CONDUCT)
```

Comment:
this link should be fine? I think only the links in CONTRIBUTING.md were broken

Reply:
It seems like I missed this while I was making the requested changes. Good catch!

Taking a look at the contributor docs on the website, they haven't been updated in over a year and do not appear to match the CNCF code of conduct. Should I address that in this PR as well or should that be done in a separate PR?

Reply:
For the website, it looks like we had copy pasted it, so we can do that to update it to the latest